### ISSUE TITLE
RXUpdater (Suggestion)

### ISSUE THREAD

**Issue by Zetchzie** [roxas75/rxTools#370]
Would a updater be possible to addin in the future so we can get the latest from our 3ds and never have to take our sd cards out?

**Comment by Syphurith** [comment]
@Zetchzie *Eh Sorry i have to tag you once more..* There is MicroSD management tool modifed for O3DS, while it can be run on N3DS more simply.

**Comment by Zetchzie** [comment]
Yeah I get that but it would be simpler to have somthing update and install for you without having to get everything from the computer. Anyways just a suggestion.

**Comment by Syphurith** [comment]
@Zetchzie Yeah i know.. Still thanks for this suggestion. If they would like to use ARM11K or HBL Homebrew to update, with WIFI.. *Wait those CFWs aren't in InstallMii repo?*
BTW this may not receive the light for quite some time, cause this is enhancement..
